### PR TITLE
updated Japanese readme

### PR DESCRIPTION
**Pull request by jbondpdx**
@LarissaLane gave me an updated Japanese readme today, so let's get it in here.

### CODE REVIEW

**Comment by LarissaLane** [comment]
Thanks Jean!

**Comment by smortex** [comment]
For your information, I just submitted #396 which fixes some spacing issues in the original `README.markdown`.  It looks like these syntax problems also exist in your proposed change, so you will probably want to fix them :wink:.

File: readmes/README_ja_JP.md [hunk 1]
```diff
@@ -139,91 +139,91 @@ NTPdの一部のバージョン(Novell DSfWなど)で、W32Timeとの互換性
...
 
-オプション
+任意
 
 データタイプ: 文字列
 
-構成ファイルのEPPテンプレートへの絶対パスまたは相対パスを指定します(値の例: 'ntp/ntp.conf.epp')。このパラメータと`config_template`パラメータの**両方**を指定すると、検証エラーがスローされます。
+構成ファイルのEPPテンプレートへの絶対パスまたは相対パスを指定します(値の例: 'ntp/ntp.conf.epp')。このパラメータと`config_template`パラメータの**両方**を指定すると、バリデーションエラーが発生します。
 
 ####`config_file_mode`
```
Comment (pmcmaw) [inline readmes/README_ja_JP.md]: Requires a space for rendering.

File: readmes/README_ja_JP.md [hunk 2]
```diff
@@ -512,27 +522,27 @@ NTP統計の保存先(ntpモニタリングが有効になっている場合)
 
 ####`step_tickers_epp`
```
Comment (pmcmaw) [inline readmes/README_ja_JP.md]: Requires a space for rendering.

**Comment by pmcmaw** [comment]
@smortex - Thank you very much for your input. 
I have added review comments to ensure we do not miss this.

**Comment by jbondpdx** [comment]
@pmcmaw , I added those spaces in, so we should be good. Thanks again to @smortex for the catch on those.

**Pull request merged**